### PR TITLE
add exclusiveLinks flag to api docs

### PR DESCRIPTION
**Pull request by ChrisCavs**

### CODE REVIEW

File: docs/tagpublisher.rst [hunk 1]
```diff
@@ -69,5 +69,10 @@ Toggle LinkMate
 
     http://amazon.com.example/BF93JSD34/ref=ods?#locklink
 
+* To make all LinkMate links on the page exclusive, you can use the `exclusiveLinks` flag on the
```
Comment (myfashionhub) [inline docs/tagpublisher.rst]: this can be moved after `donotlink`

**Comment by myfashionhub** [comment]
Need to bump version as well

File: docs/tagpublisher.rst [hunk 2]
```diff
@@ -61,6 +61,12 @@ Toggle LinkMate
 
     window.NRTV_EVENT_DATA.donotlink = true;
 
+* To make all LinkMate links on the page exclusive, you can use the `exclusiveLinks` flag on the
```
Comment (csdev) [inline docs/tagpublisher.rst]: use double backticks for inline code in sphinx/rst

File: docs/tagpublisher.rst [hunk 3]
```diff
@@ -61,6 +61,12 @@ Toggle LinkMate
 
     window.NRTV_EVENT_DATA.donotlink = true;
 
+* To make all LinkMate links on the page exclusive, you can use the ``exclusiveLinks`` flag on the
+  Narrativ window object
+  ::
+
+    window.NRTV_LINKMATE_DATA = {exclusiveLinks: true};
```
Comment (csdev) [inline docs/tagpublisher.rst]: is there a reason you are not specifying this as `window.NRTV_LINKMATE_DATA.exclusiveLinks`, following the format used in the previous items?

Comment (ChrisCavs) [reply docs/tagpublisher.rst]: the NRTV_LINKMATE_DATA object is created after the links are retrieved.  using `window.NRTV_LINKMATE_DATA.exclusiveLinks` would result in a typeError

**Pull request merged**